docs/supported_alpha.md

```diff
@@ -0,0 +1,1141 @@
+﻿---
+id: supported_alpha
+title: Functions Sorted Alphabetically
+sidebar_label: Functions - Alpha Sort
```

Comment:
Maybe `Support Matrix` or `Support Table`, as it contains unsupported functions?